### ISSUE TITLE
Include diagnostics in tap output

### ISSUE THREAD

**Issue by hisuwh** [alsatian-test/alsatian#750]
The TAP specification includes a standard for diagnostics (i.e. file and line number): https://node-tap.org/tap-protocol/#yaml-diagnostics

Could these be included in the alsatian tap output?

**Comment by jamesadarich** [comment]
Yep absolutely, there's work outstanding on this so will make sure it is to spec. Line number isn't there yet but I'm sure we can get that in too :)

**Comment by hisuwh** [comment]
Happy to try and help with this one though not sure when I will have time.

**Comment by jamesadarich** [comment]
Awesome will be much appreciated
